helm_deploy/values-dev.yaml

```diff
@@ -5,7 +5,7 @@ generic-service:
   replicaCount: 2
 
   ingress:
-    host: contacts-ui-dev.hmpps.service.justice.gov.uk
+    host: contacts-dev.hmpps.service.justice.gov.uk
```

Comment:
Could you make same change as this to the -preprod and -prod values too please? We dont actually need right now, but it will catch us out later if we don't. Thanks.

Reply:
I will make a note of that, im sure i will forget otherwise :)